src/components/LinkSection.jsx

```
@@ -0,0 +1,25 @@
+
+const LinkSection = ({ lable, data }) => {
+  return (
+    <div className=" flex flex-col gap-3">
+      <div className="flex flex-col gap-2">
+        <p className="p-bold-20 text-grey-600">{lable}: </p>
+        {/* <p className="p-medium-26 lg-p-regular-18">{edir?.description}</p> */}
+      </div>
+
+      <div className="flex flex-wrap gap-6">
+        {data.map((data) => (
+          <Link to={`/${lable.toLowerCase()}/${data.title | data.name}}`} className="flex gap-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)